forge/build.gradle

```diff
@@ -77,6 +77,13 @@ dependencies {
     modImplementation("com.jozufozu.flywheel:flywheel-forge-${flywheel_minecraft_version}:${flywheel_version_forge}")
     modImplementation("com.tterrag.registrate:Registrate:${registrate_version_forge}")
 
+    modImplementation("net.createmod.ponder:Ponder-Forge-${minecraft_version}:${ponder_version}")
+    modImplementation("dev.engine-room.flywheel:flywheel-forge-api-${minecraft_version}:${flywheel_version}")
+    modImplementation("dev.engine-room.flywheel:flywheel-forge-${minecraft_version}:${flywheel_version}")
+    modImplementation(annotationProcessor("io.github.llamalad7:mixinextras-common:0.4.1"))
```

Comment:
Hard coded versions could be made a build property

Reply:
Fixed